src/Workspaces/Core/Portable/LanguageServices/SyntaxFactsService/AbstractSyntaxFactsService.cs

```diff
@@ -571,5 +571,8 @@ public bool IsUsingStatement(SyntaxNode node)
 
         public bool IsReturnStatement(SyntaxNode node)
             => node.RawKind == SyntaxKinds.ReturnStatement;
+
+        public bool IsExpressionStatement(SyntaxNode node)
+            => node.RawKind == SyntaxKinds.ExpressionStatement;
```

Comment:
i would love if we could make all these `Is` functions handle the `null` case by using `?.RawKind ==`.  It's frustrating me that we have a mismash of each style :)

Reply:
I'll consider making sweep for all of these functions if I get blessing to introduce `#nullable` to parts of the service.